server/app/featureflags/FeatureFlags.java

```diff
@@ -32,11 +32,20 @@ public boolean areOverridesEnabled() {
         && config.getBoolean(FEATURE_FLAG_OVERRIDES_ENABLED);
   }
 
-  /** If the Status Tracking feature is enabled. */
+  /**
+   * If the Status Tracking feature is enabled.
+   *
+   * <p>Allows for overrides set in {@code request}.
+   */
```

Comment:
High-level, can we add a browser smoke test for these or is that overkill? I mainly ask because we already have testing for program admin exports that the CSV content includes / excludes certain applications. Presumably we could add an additional assertion that the status is set appropriately in the response (e.g. admin_application_statuses.test.ts.

Reply:
Hmmmmm. I think so given that I recently added /dev/feature.  Currently it only prints the general configuration for overrides, but I was planning on adding the actual feature flags values too.

```
Overrides are allowed if all are true:
Server environment: true
Configuration: true
```


Are you OK with that in another PR?

Reply:
Works for me.